CHANGELOG.rst

```diff
@@ -1,6 +1,19 @@
 Changelog
 =========
 
+Version v1.2.0
+--------------
+
+Breaking Changes
+~~~~~~~~~~~~~~~~
+- ``nodes.get`` and ``edges.get`` (and ``network.get``) no longer return a dataframe
+  - returns a generator yielding tuples of ``(<population_name>, <dataframe>)`` instead
```

Comment:
maybe add a comment about how one can take that generator, and get the previous behavior?

Reply:
Added this in changelog and also added some examples in  notebooks (previously not covered) that gives a hint how to  work with `nodes.get` and `edges.get`.

Reply:
great, thanks